internal/seleniumtest/seleniumtest.go

```diff
@@ -1700,7 +1700,29 @@ func testChromeExtension(t *testing.T, c Config) {
 	}
 }
 
+func testChromeCdp(t *testing.T, c Config) {
```

Comment:
Change this method name to exactly mirror the method name being tested, "testExecuteChromeDPCommand" (see below for a discussion on the method name).